sogs/routes/rooms.py

```diff
@@ -960,3 +960,37 @@ def delete_all_posts(room, sid):
     if not deleted:
         abort(http.NOT_FOUND)
     return jsonify({})
+
+
+@rooms.delete("/rooms/all/<SessionID:sid>")
+def delete_user_posts_from_all_rooms(sid):
+    """
+    Deletes all posts from all rooms by a given user.
+
+    # URL Parameters
+
+    - `sid` — the session id of the user to ban
+
+    # Return value
+
+    A JSON dict with the keys:
+
+    - `total` — The total number of posts deleted across all rooms.
+    - `rooms` — A dict of room tokens and their deletion counts.
+
+    # Error status codes
+
+    - 403 Forbidden — if the invoking user does not have access to a room.
```

Comment:
I don't think this error can happen from here anymore; I think if you don't have any access you'll just get 0 deletions.

Reply:
Yeah, I wondered myself about that.

Removed.

Reply:
Looks like this only got half-removed.

Reply:
A good thing one of us is paying attention.